provider/pkg/gen/nodejs-templates/helm/v2/helm.ts

```diff
@@ -361,6 +367,8 @@ interface ResolvedFetchOpts {
  * If the `verify` option is specified, the requested chart MUST have a provenance file, and MUST
  * pass the verification process. Failure in any part of this will result in an error, and the chart
  * will not be saved locally.
+ *
+ * @ignore
```

Comment:
What does this do?

Reply:
It's supposed to ignore it in the generated docs since it's an internal function